### PR TITLE
0.12'ify

### PR DESCRIPTION
**Pull request by esacteksab**
* 0.12 TF support
* Terraform011 branch created.
* Terratest testing basic functionality.

### CODE REVIEW

File: .terraform-version [hunk 1]
```diff
@@ -1 +1 @@
-0.11.14
+0.12.18
```
Comment (dynamike) [inline .terraform-version]: We can delete this file now that we aren't using 0.11 anywhere.

File: examples/simple/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ---------------------------------------------------------------------------------------------------------------------
```
Comment (dynamike) [inline examples/simple/outputs.tf]: Not needed

File: examples/simple/variables.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# ---------------------------------------------------------------------------------------------------------------------
```
Comment (dynamike) [inline examples/simple/variables.tf]: Strip out the comments

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2019, TrussWorks, Inc.
+Copyright (c) 2018, TrussWorks, Inc.
```
Comment (esacteksab) [inline LICENSE]: Should this now be 2020?

Comment (dynamike) [reply LICENSE]: it should :)

**Pull request merged**